### DISCUSSION TITLE
【广告】白祇重工招聘

### DISCUSSION THREAD

**Discussion by Belobog-Official** [share121/inter-knot#1482] [其他]
本次招聘3名机械工程师、5名试验调试工。包吃不包住，薪资面议。有经验者优先。
白祇重工可能会多次进行空洞内作业。请各位量力而行。

**Comment by reversed-PAI** [comment]
试验调试工会有危险吗？我听说白祇重工的机器都不太稳定。。。会不会伤人啊？

**Comment by gezhao96** [comment]
我我我，招我！

**Comment by Belobog-Official** [comment]
回一楼：白祇重工的机器在格莉丝手中还是很有保障的。大家担心的原型机信号问题也已经解除。（附：感谢法厄同）

**Comment by EhuanyingK** [comment]
机器人在格莉丝手中有保障？狡兔屋的比利能来吗

**Comment by Belobog-Official** [comment]
机器人的话……倒是可以。
但一边工作一边念星徽骑士台词的机器人还是算了

**Comment by reversed-PAI** [comment]
有格莉丝大姐姐吗！我来！😍
